hs_core/views/__init__.py

```diff
@@ -64,6 +65,33 @@
 logger = logging.getLogger(__name__)
 
 
+class HydroshareWebServices():
+
+    def __init__(self, services_url, api_token, timeout):
+        self.services_url = services_url
+        self.timeout = timeout
+        self.session = requests.Session()
+        self.session.headers.update(
+            {"Authorization": " ".join(("Token", str(api_token)))}
+        )
+
+    def update(self, res_id):
+        rest_url = str(self.services_url) + "/update-services/?res_id=" + str(res_id)
+        try:
+            return self.session.post(rest_url, timeout=self.timeout)
+        except requests.exceptions.RequestException as requests_exception:
+            return requests_exception
+        except ValueError as requests_exception:
+            return requests_exception
+
+
+web_services = HydroshareWebServices(
+    services_url="http://localhost:8080/apps/hydroshare-web-services-manager/api",
+    api_token="932aebc70b5924dd31da63f8d80e7c62db18c02c",
+    timeout=20
+)
```

Comment:
It'd be good to move this `HydroShareWebServices` class into a utility module, such as https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/utils.py so that it can be used by other modules easily for better encapsulation.